src/wp-content/themes/twentytwentyfive/patterns/cta-centered-heading.php

```diff
@@ -25,7 +25,7 @@
 		<!-- wp:buttons {"layout":{"type":"flex","justifyContent":"center"}} -->
 		<div class="wp-block-buttons">
 			<!-- wp:button -->
-			<div class="wp-block-button"><a class="wp-block-button__link wp-element-button"><?php esc_html_e( 'Learn more', 'twentytwentyfive' ); ?></a></div>
+			<div class="wp-block-button"><a class="wp-block-button__link wp-element-button"><?php esc_html_e( 'Learn more', 'Main CTA button text.', 'twentytwentyfive' ); ?></a></div>
```

Comment:
Contexts can not be added to `esc_html_e()`. Please see the reported error below.